apps/filecoin-site/src/app/_components/Footer/NewsletterForm.tsx

```diff
@@ -0,0 +1,28 @@
+import { ArrowRightIcon } from '@phosphor-icons/react/dist/ssr'
+
+export function NewsletterForm() {
+  return (
+    <form className="flex min-w-[100px] flex-col gap-2 text-white">
+      <label htmlFor="newsletter-email" className="text-sm font-medium">
+        Sign up for Filecoin updates
+      </label>
+      <div className="relative flex items-center gap-4">
+        <input
+          id="newsletter-email"
+          type="email"
+          placeholder="Your email"
+          className="h-13 w-full border-0 border-b border-white bg-transparent placeholder-zinc-400 placeholder:text-sm focus:border-white focus:ring-0"
+          autoComplete="email"
+        />
+        <button
+          type="submit"
+          className="absolute right-0 -mr-3 flex h-12 w-12 cursor-pointer items-center justify-center"
+        >
+          <div className="gradient-svg-fill">
```

Comment:
Is this `gradient-svg-fill` going to be use anywhere else? Is it a brand style? Can we inline the styles?